react/features/recent-list/functions.js

```diff
@@ -31,6 +31,37 @@ require('moment/locale/zh-cn');
 import { i18next } from '../base/i18n';
 import { parseURIString } from '../base/util';
 
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+/**
+ * The name of the {@code window.localStorage} item where recent rooms are
+ * stored.
+ *
+ * @type {string}
+ */
+const RECENT_URL_STORAGE = 'recentURLs';
+
+/**
+ * Retrieves the recent room list that was stored using the legacy way.
+ *
+ * @returns {Array<Object>}
+ */
+export function getLegacyRecentRoomList(): Array<Object> {
```

Comment:
A. A feature's functions.js is supposed to contain functions which the feature (1) itself invokes in more than one file or (2) exports for other features to invoke as well. I don't see that to be the case `getLegacyRecentRoomList` so I'm curious whether there's an intention to reuse the function in the future (which seems unlikely to me but...)